app/uk/gov/hmrc/apiplatformmicroservice/thirdpartyapplication/services/UpliftApplicationService.scala

```diff
@@ -70,7 +75,7 @@ class UpliftApplicationService @Inject() (
         remappedRequestSubs        = CdsVersionHandler.adjustSpecialCaseVersions(requestedApiSubs)
         filteredSubs               = remappedRequestSubs.filter(upliftableApis.contains)
         _                         <- cond(filteredSubs.nonEmpty, (), "Request contains apis that cannot be uplifted")
-        filteredUpliftRequest      = upliftRequest.copy(subscriptions = filteredSubs)
+        filteredUpliftRequest      = upliftRequest.copy(subscriptions = Set.empty)
```

Comment:
just checking you meant to do this?

Reply:
Yes.  Temporarily until we move stuff from APM into TPA